Updating emergency policy cmdlets (#9677)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Set-CsTeamsEmergencyCallingPolicy.md

### Identity (Default)
```
Set-CsTeamsEmergencyCallingPolicy [[-Identity] <string>] [-Description <string>] [-EnhancedEmergencyServiceDisclaimer <string>]
 [-ExternalLocationLookupMode <string>] [-NotificationDialOutNumber <string>] [-NotificationGroup <string>] [-NotificationMode <Object>]
 [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION
This cmdlet modifies an existing Teams Emergency Calling policy. Emergency calling policy is used for the life cycle of emergency calling experience for the security desk and Teams client location experience.

## EXAMPLES

### Example 1
```powershell
Set-CsTeamsEmergencyCallingPolicy -Identity "testECP" -NotificationGroup "123@gh.com;567@test.com"
```

> This example modifies a existing cmdlet with identity testECP.

Reviewer: ```suggestion
This example modifies an existing cmdlet with identity testECP.
```

Accept wildcard characters: False
```

### -ExternalLocationLookupMode

> Enable ExternalLocationLookupMode. This mode allow users to set Emergency addresses for remote locations.

Reviewer: ```suggestion
Enables ExternalLocationLookupMode. This mode allows users to set Emergency addresses for remote locations.
```

Accept wildcard characters: False
```

### -NotificationDialOutNumber

> This parameter represents PSTN number which can be dialed out if NotificationMode is set to either of the two Conference values

Reviewer: ```suggestion
This parameter represents the PSTN number which can be dialed out if NotificationMode is set to either of the two Conference values.
```
